### PR TITLE
feat: admin settings design update

### PR DESCRIPTION
**Pull request by shohag121**
### All Submissions:

* [x] My code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
* [x] My code satisfies feature requirements
* [x] My code is tested
* [x] My code passes the PHPCS tests
* [x] My code has proper inline documentation
* [x] I've included developer documentation (optional)
* [x] I've added proper labels to this pull request

### Changes proposed in this Pull Request:

Admin Settings Re design

* Closes https://github.com/weDevsOfficial/dokan-pro/issues/2500 .

### How to test the changes in this Pull Request:

1. check https://github.com/weDevsOfficial/dokan-pro/issues/2500

### Changelog entry

No Change log is needed.

### CODE REVIEW

File: src/admin/pages/Settings.vue [hunk 1]
```diff
@@ -609,11 +609,11 @@
 
         div.nav-tab-wrapper {
             width: 340px;
-            border: 1px solid #c8d7e1;
+            //border: 1px solid #c8d7e1;
```
Comment (nurul-umbhiya) [inline src/admin/pages/Settings.vue:612]: kindly remove all the commented out code if you don't need this code anymore. You'll be able to retrieve old code vai git if needed in the future.

**Review by nurul-umbhiya — CHANGES_REQUESTED** [review]

File: src/admin/pages/Settings.vue (hunk 1 above)
Comment (shohag121) [reply src/admin/pages/Settings.vue]: Removed bhai.

**Comment by shashwatahalder01** [comment]
Scenario:
Marked button border color should be consistent with the rest of the design
 
Screenshot at Jul 27 08-58-57

**Comment by shohag121** [comment]
@shashwatahalder01 bhai, Issues are taken care of.

**Review by nurul-umbhiya — APPROVED** [review]

**Pull request merged**